runtime/src/bpftime_shm_internal.cpp

```diff
@@ -41,9 +41,11 @@ extern "C" void bpftime_destroy_global_shm()
 extern "C" void bpftime_remove_global_shm()
 {
 	using namespace bpftime;
-	boost::interprocess::shared_memory_object::remove(
-		get_global_shm_name());
-	SPDLOG_INFO("Global shm removed");
+	const char *shm_name = get_global_shm_name();
+	boost::interprocess::shared_memory_object::remove(shm_name);
+	if (shm_name != DEFAULT_GLOBAL_SHM_NAME) {
```

Comment:
Why compare c-string in this way?

Reply:
I'll correct it in the next commit with strcmp